scripts/barclamp_mgmt_lib.rb

```diff
@@ -67,6 +67,20 @@ def fatal(msg, log = nil, exit_code = 1)
   exit exit_code
 end
 
+def get_crowbar_yml_path(directory, suggested_bc_name = nil)
+  if suggested_bc_name.nil?
+    suggested_bc_name = directory.split(File::SEPARATOR)[-1]
+  end
+
+  # Look for both old name (crowbar.yml) and new name ($barclamp.yml)
+  ["#{suggested_by_name}.yml", "crowbar.yml"].each do |basename|
+    path = File.join(directory, basename)
+    return path if File.exists?(path)
+  end
+
+  nil
```

Comment:
You're correct, but I prefer to be explicit about it to make it clear that it's intended. The alternative is to put a comment, but then it's the same amount of lines :-)